VAMobile/src/store/api/demo/mocks/appointments.json

```diff
@@ -96,13 +96,13 @@
               "id": null,
               "name": "Outpatient Clinic",
               "address": {
-                "street": "2341 North Ave",
-                "city": "Commerce",
+                "street": "11201 Benton Street",
+                "city": "Loma Linda",
```

Comment:
This is my local VA facility 🙂

Reply:
Nice, I went to the one in Cheyenne, WY for my PIV card. That one shows up in demo mode as well